### PR TITLE
Update SQL spec descriptions

### PR DESCRIPTION
**Pull request by jaredmoo**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jaredmoo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaredmoo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-sql/2014-04-01/swagger/firewallRules.json [hunk 1]
```diff
@@ -241,7 +241,7 @@
           "items": {
             "$ref": "#/definitions/ServerFirewallRule"
           },
-          "description": "The list of Azure SQL server firewall rules for the server."
+          "description": "The list of server firewall rules for the server."
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/firewallRules.json]: Can you just make it "The list of firewall rules for the server" for this one?

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "Azure SQL Database API spec",
-    "description": "Provides create, read, update and delete functionality for Azure SQL resources including servers, databases, elastic pools, recommendations, operations, and usage metrics.",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: This is the one place I'd like to leave Azure SQL

File: arm-sql/2014-04-01/swagger/firewallRules.json (hunk 1 above)
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/firewallRules.json]: Can you just make it "The list of firewall rules for the server" for this one?

File: arm-sql/2014-04-01/swagger/sql.core.json (hunk 2 above)
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: This is the one place I'd like to leave Azure SQL in

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 3]
```diff
@@ -294,7 +294,7 @@
           "Databases"
         ],
         "operationId": "Databases_ListByServer",
-        "description": "Returns information about an Azure SQL database.",
+        "description": "Returns information about a database.",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: I know this isn't your error, but could you put "Returns a list of databases by server."?

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 4]
```diff
@@ -375,7 +375,7 @@
           "Servers"
         ],
         "operationId": "Servers_List",
-        "description": "Returns information about an Azure SQL server.",
+        "description": "Returns information about a server.",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: "Returns a list of servers."?

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 5]
```diff
@@ -554,7 +554,7 @@
           "Servers"
         ],
         "operationId": "Servers_ListUsages",
-        "description": "Returns information about Azure SQL server usage.",
+        "description": "Returns information about server usage.",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: usages

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 6]
```diff
@@ -943,7 +943,7 @@
           "RecommendedElasticPools"
         ],
         "operationId": "RecommendedElasticPools_Get",
-        "description": "Gets information about an Azure SQL Recommended Elastic Pool.",
+        "description": "Gets information about an recommended elastic pool.",
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: a recommended

File: arm-sql/2014-04-01/swagger/sql.core.json [hunk 7]
```diff
@@ -1560,7 +1560,7 @@
           "$ref": "#/definitions/Resource"
         }
       ],
-      "description": "Represents an Azure SQL Recommended Elastic Pool."
+      "description": "Represents an recommended elastic pool."
```
Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: Maybe ctrl+replace 'an recommended'

Comment (nathannfan) [inline arm-sql/2014-04-01/swagger/sql.core.json]: Maybe ctrl+replace 'an recommended'

**Review by nathannfan — APPROVED** [review]

**Review by anuchandy — COMMENTED** [review]
CI is failing specifically with error "FATAL: Error generating client model: System.InvalidOperationException: Swagger document contains two or more x-ms-enum extensions with the same name 'ServiceObjectiveName' and different values.", please take a look and fix.

**Comment by jaredmoo** [comment]
Fixed, thanks. Node build seems to be hitting a transient error.

**Comment by anuchandy** [comment]
@jaredmoo thanks, restarted job and now it is passing.

**Comment by jaredmoo** [comment]
Can I get a review and merge here?

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
@jaredmoo thanks, merging.

**Comment by jaredmoo** [comment]
Thanks!

**Pull request merged**